indico/modules/events/models/persons.py

```diff
@@ -244,7 +262,7 @@ def merge_person_info(self, other):
             if existing_event_link is None:
                 event_link.person = self
             else:
-                db.session.delete(event_link)
+                other.event_links.remove(event_link)
```

Comment:
like this it's consistent with what we have in contributions, sessions, etc. also, you can override things on the link level. with just a column this wouldn't be possible